server/src/schemas/usersSchema.ts

```diff
@@ -0,0 +1,73 @@
+import { gql } from "apollo-server-express";
+
+export const UsersTypeDefs = gql`
+  type User {
+    id: ID!
+    first_name: String!
+    last_name: String!
+    email: String!
+    type: Type!
+    privilege: Privilege!
+    registration_date: Date!
+    balance: Int!
+    hold: Boolean!
+    holds: [Hold]
```

Comment:
What's the point of `hold` when we have `holds`?

Reply:
I guess you could just check if holds is empty yeah